controllers/rest/api.go

```diff
@@ -293,6 +295,11 @@ func (p *RESTAPI) GetEntityFactories() map[string]model.EntityFactory {
 	return p.entityFactories
 }
 
+//GetSessionStore get gorm session store
+func (p *RESTAPI) GetSessionStore() *gormstore.Store {
```

Comment:
to accommodate different stores it would be more ideal if this expected the Store interface https://github.com/gorilla/sessions/blob/master/store.go#L21